expe.py

```diff
@@ -0,0 +1,5 @@
+#This defines the structure of the experiment
+
+
+class Expe(object):
+    n = 10
```

Comment:
For various historical reasons, it's good to have a linebreak at the end all code files (http://stackoverflow.com/questions/729692/why-should-files-end-with-a-newline)